server/apps/immich/src/api-v1/asset/asset.service.ts

```diff
@@ -456,7 +456,7 @@ export class AssetService {
 
         await fs.access(videoPath, constants.R_OK | constants.W_OK);
 
-        if (query.isWeb && asset.mimeType == 'video/quicktime') {
+        if (query.isWeb && asset.mimeType == 'video/quicktime' || query.isWeb && asset.mimeType == "video/x-msvideo") {
```

Comment:
Is this change related to this PR?

Reply:
Yes, because previously all files went to get transcoded to MP4 and this PR changed it but .avi files need the transcode as well due to browser support not existing.

Reply:
I think this is a good catch - there's basically a discrepancy between when we transcode a video and when we use the transcoding. Can you make a new function `isWebPlayable(mimeType: string)`. Maybe we can put it in `asset-upload.config.ts` for now, which returns true for all types besides `video/mp4`? And then use that function here (`asset.service.ts`) as well as in the mp4 processor?

Reply:
I have now moved it to a central location and transcoder uses that too.